### PR TITLE
fixes and improvements

### PR DESCRIPTION
**Pull request by slively**
remove Dockerfile (this was a bad idea on my part, we can talk Tuesday about it)
add docker login
add shell script for one line builds
skip temp dir if only a single docker context
fix yaml.load warning
fix requirements.txt

### CODE REVIEW

**Comment by slively** [comment]
It ends out being a one liner in the build config: https://github.com/MVPStudio/emerald-citizen/blob/build-stuff/.circleci/config.yml#L29

File: build_tag_push.py [hunk 1]
```diff
@@ -28,28 +30,35 @@ def parse_app_file(filename):
     for the yaml file.
     '''
     with open(filename, "r") as file_contents:
-        dict_of_contents = yaml.load(file_contents)
+        dict_of_contents = yaml.load(file_contents, Loader=yaml.FullLoader)
 
     return dict_of_contents
 
+def docker_login():
+    subprocess.check_call(["docker", "login", "-u" ,os.environ[str('DOCKER_LOGIN')], "-p", os.environ[str('DOCKER_PWD')]])
```
Comment (oliverdain) [inline build_tag_push.py:38]: pretty sure this line is too long for PEP8. Did you run pep8/flake8 over it?

File: build_tag_push.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+git clone --depth 1 -b master https://github.com/MVPStudio/build_setup.git ~/build_setup
+ls -l ~/build_setup
```
Comment (oliverdain) [inline build_tag_push.sh:2]: what's the purpose of this? Just for logging/debugging?

File: build_tag_push.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+sudo pip install -r ~/build_setup/requirements.txt
+python ~/build_setup/build_tag_push.py
```
Comment (oliverdain) [inline build_tag_push.sh:4]: newline at end of file

**Review by oliverdain — APPROVED** [review]
Looks really good. A few minor nits. Will let @alyssakelley do the final approve since it's mods to her code.

**Pull request merged**